notebook/oai_chatgpt_gpt4.ipynb

```diff
@@ -17,6 +17,11 @@
     }
    },
    "source": [
+    "Contributions to this project, i.e., https://github.com/autogen-ai/autogen, are licensed under the Apache License, Version 2.0 (Apache-2.0).\n",
+    "Copyright (c) 2023 - 2024, Owners of https://github.com/autogen-ai\n",
+    "SPDX-License-Identifier: Apache-2.0\n",
+    "Portions derived from  https://github.com/microsoft/autogen under the MIT License..\n",
+    "SPDX-License-Identifier: MIT\n",
     "Copyright (c) Microsoft Corporation. All rights reserved. \n",
     "\n",
     "Licensed under the MIT License.\n",
```

Comment:
also here

Reply:
Is the comment addressed? The general principle is do not remove existing licenses.

Reply:
yes let me solve this. thanks